test/ruby/test_compile_prism.rb

```diff
@@ -849,6 +849,26 @@ def test_RescueNode
       assert_prism_eval("begin; rescue; end")
     end
 
+    def test_RetryNode
```

Comment:
Can we add a few more test cases? Specifically with empty begins, rescues with just a retry without anything else in the rescue?